src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -160,60 +160,64 @@ private AdsDataProvider() {
 
   @DataProvider
   public static Object[][] skin() {
-    final String wikiName = "project43";
-    final String pageName = "SyntheticTests/Skin";
+    final String noSkinWikiName = "adtest";
+    final String noSkinPageName = "Skin";
+
+    final String gradientWikiName = "adtest-pluto";
+    final String gradientPageName = "Skin";
```

Comment:
Could you write comment instead of setting additional info in variable name?
Could you move variable in class scope and rename all the occurrences?